net-misc/pcapfix/pcapfix-1.1.4.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Repair and fix broken pcap files"
+HOMEPAGE="http://f00l.de/pcapfix/"
+SRC_URI="http://f00l.de/pcapfix/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+RDEPEND=""
+DEPEND="${RDEPEND}"
+
+DOCS=( Changelog README )
+
+src_prepare() {
+	default
+	sed -e 's/gcc/$(CC) $(CFLAGS)/g' -i Makefile
+	use debug || sed -e 's/DEBUGFLAGS = -g/DEBUGFLAGS =/g' -i Makefile
```

Comment:
This also needs to be appended with `|| die`. With EAPI-7, `(use debug || sed -e 's/DEBUGFLAGS = -g/DEBUGFLAGS =/g' -i Makefile) || die` should work, but if it doesn't, use `if use debug ; then` form instead.